### ISSUE TITLE
Toggling comment using Ctrl + / not work

### ISSUE THREAD

**Issue by foo2810** [foo2810/ad-hoc-vimconfig#1]
**Vesrion**
vim v8.0.1763

**Details**
After selecting a range in visual mode, pressing Ctrl + / moves the cursor to the beginning of the selected range without commenting out (or uncommenting) it.

Trimming trailing white space also does not work.

**Frequency**
every time

**Comment by foo2810** [comment]
`<Cmd>` key does not work as expected.

For example
```vim
vmap <C-_> <esc>`<<Cmd>call g:Toggle_Comment()<CR>
```

```vim
call feedkeys("_\<Cmd>call g:Toggle_Comment_Iter()\<CR>j\<esc>")
```

**Comment by foo2810** [comment]
Perhaps this patch of vim is related. This patch introduced `<Cmd>` key to vim.

https://github.com/vim/vim/commit/957cf67d50516ba98716f59c9e1cb6412ec1535d
https://github.com/vim/vim/pull/7282

Toggling comments may work correctly in vim v8.2.1978.

**Comment by foo2810** [comment]
Check the vim version in which the feature works

vim v8.2.1978: work
vim v8.2.1977: not work

**Comment by foo2810** [comment]
To support a wide range of vim versions, a method that does not use <Cmd> should be considered.

If `<Cmd>` or `\<Cmd>` is replaced by `:`, it may work well.

**Comment by foo2810** [comment]
This bug was fixed in 8cdeb093c27d9cbfae3fe5df96b1fb5fd3b3be18
